### PR TITLE
Improve usage and add usage example with comments

### PR DESCRIPTION
**Pull request by Briksins**
Please review my changes

Possibly you would want to add them to your main code

### CODE REVIEW

**Comment by Over17** [comment]
@Briksins thanks for your contribution - looks good! Like the delegates.
Some minor issues/typos are in place, after that we can merge it!

**Comment by Briksins** [comment]
Oh sorry, seen your reply while driving and completely forgot to come back to you.

Any chance you could point exactly what you want me to fix? and ill update it tomorrow from work (code is in work PC)

**Comment by Over17** [comment]
@Briksins I've added the comments to the pull request - please let me know if you have any issues seeing them.

**Comment by Over17** [comment]
Sorry didn't click the button "submit review" - thought it was automatic.

File: Assets/Scripts/AndroidPermissionsManager.cs [hunk 1]
```diff
@@ -1,19 +1,42 @@
+using System;
 using UnityEngine;
 
 public class AndroidPermissionCallback : AndroidJavaProxy
 {
-    public AndroidPermissionCallback() : base("com.unity3d.player.UnityAndroidPermissions$IPermissionRequestResult") { }
+	private event Action<string> OnPermissionGrantedAction;
```
Comment (Briksins) [inline Assets/Scripts/AndroidPermissionsManager.cs]: Unfortunatly work policy are tabs, everything setup to be with tabs, I tried to find those settings in VS, but VS settings are nightmare, so it's all tabs as before, hope you will accept it with tabs

All other misspells which I found and you found were fixed

Comment (Over17) [inline Assets/Scripts/AndroidPermissionsManager.cs]: Just that you know: https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/ :)

**Comment by Over17** [comment]
Thanks for your contribution! I'll reformat things myself.

**Pull request merged**